packages/gafl-webapp-service/src/services/analytics/analytics.js

```diff
@@ -0,0 +1,15 @@
+
+export const getTrackingProductDetailsFromTransaction = transaction =>
+  transaction.permissions.map(permission => ({
```

Comment:
I know I didn't use this in the sample code but on reflection it might be nicer to use object destructing here to change from:
`transaction.permissions.map(permission => ({`
to
`transaction.permissions.map( { permit } => ({`

as all code below does permission.permit to access the permit object.